src/EA.Weee.RequestHandlers.Tests.Unit/Admin/Obligations/GetSchemesWithObligationHandlerTests.cs

```diff
@@ -0,0 +1,122 @@
+﻿namespace EA.Weee.RequestHandlers.Tests.Unit.Admin.Obligations
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using AutoFixture;
+    using Core.Scheme;
+    using DataAccess.DataAccess;
+    using Domain.Scheme;
+    using FakeItEasy;
+    using FluentAssertions;
+    using Prsd.Core.Mapper;
+    using RequestHandlers.Admin.Obligations;
+    using RequestHandlers.Security;
+    using Requests.Admin.Obligations;
+    using System.Security;
+    using System.Threading.Tasks;
+    using Weee.Tests.Core;
+    using Xunit;
+
+    public class GetSchemesWithObligationHandlerTests : SimpleUnitTestBase
```

Comment:
can you add **unit** to the file name and class, it's duplicated with the  EA.Weee.Integration.Tests.Handlers GetSchemesWithObligationHandlerTests name

Reply:
done...